package.json

```diff
@@ -9,26 +9,32 @@
   "license": "MIT",
   "repository": "expressjs/vhost",
   "devDependencies": {
+    "@types/express": "4.11.1",
+    "@types/node": "9.6.5",
     "eslint": "3.16.1",
     "eslint-config-standard": "6.2.1",
     "eslint-plugin-markdown": "1.0.0-beta.3",
     "eslint-plugin-promise": "3.4.2",
     "eslint-plugin-standard": "2.0.1",
     "istanbul": "0.4.5",
     "mocha": "2.5.3",
-    "supertest": "1.1.0"
+    "supertest": "1.1.0",
+    "typescript": "2.8.1"
   },
   "files": [
     "LICENSE",
     "HISTORY.md",
-    "index.js"
+    "index.js",
+    "typings/index.d.ts"
```

Comment:
In other projects I've gotten typings, they are just in the root as `index.d.ts`. Any reason they need to be in their own folder, or can they just be in the root? A major benefit of it being in the root means that you don't need to have the `typings` key at all (just like `main` is not needed when the entry point is `index.js`).

Reply:
Sure, where would you put the test file ? `test/test.ts` sound good ?

Reply:
Test file in `test` folder makes sense to me 👍